atlite/wind.py

```diff
@@ -90,3 +101,105 @@ def extrapolate_wind_speed(ds, to_height, from_height=None):
                           "units" : "m s**-1"})
 
     return wnd_spd.rename(to_name)
+
+def download_turbineconf(turbine=None, store_locally=True):
+    """Download a windturbine configuration from the OEDB database.
+    
+    Download the configuration of a windturbine model from the OEDB database
+    into the local 'windturbine_dir'.
+    The OEDB database can be viewed here:
+    https://openenergy-platform.org/dataedit/view/supply/turbine_library
+    (2019-07-22)
+    Only one turbine configuration is downloaded at a time, if the 
+    search parameters yield an ambigious result, not data is downloaded.
+    
```

Comment:
Mention in the docs that one can get a table of all turbines at
https://openenergy-platform.org/dataedit/view/supply/turbine_library

Reply:
Duh, ok, that is already in the doc string. Sorry for the noise.